lib/bpf/common_test.go

```diff
@@ -74,49 +76,61 @@ func TestMain(m *testing.M) {
 // TestCheckAndSetDefaults makes sure defaults are set when the user does not
 // provide values for the page sizes and hard coded values (like zero or a
 // specific page size) are respected when given.
-func TestCheckAndSetDefaults(t *testing.T) {
-	var perfBufferPageCount = defaults.PerfBufferPageCount
-	var openPerfBufferPageCount = defaults.OpenPerfBufferPageCount
-	var zeroPageCount = 0
+func TestBPFConfig_CheckAndSetDefaults(t *testing.T) {
+	perfBufferPageCount := defaults.PerfBufferPageCount
+	openPerfBufferPageCount := defaults.OpenPerfBufferPageCount
+	zeroPageCount := 0
+	udpSilencePeriod := defaults.UDPSilencePeriod
+	udpSilenceBufferSize := defaults.UDPSilenceBufferSize
+	customUDPSilencePeriod := 1 * time.Minute
+	customUDPSilenceBufferSize := 42
 
 	var tests = []struct {
-		inConfig  *servicecfg.BPFConfig
```

Comment:
Note: this is mostly a refactor of the table so it uses names tests, t.Run and Go idioms (got/want). Tests now diff the entire struct, which catches additions of new defaults.